src/App.css

```diff
@@ -385,20 +385,22 @@ Footer {
   width: 100%;
   padding: 1rem 0;
   text-align: center;
-  margin: 2rem 0 1rem 0;
+  margin: 2rem 0 3rem 0;
   display: block;
+  border: none;
 }
 
 /* container with grey circle background*/
 .container-circle {
   background-image: url(assets/circle_grey.svg);
-  background-position: 100px -20px; /*TODO: find rem equivalent*/
+  background-position: 100px -50px; /*TODO: find rem equivalent*/
   background-repeat: no-repeat;
-  background-size: 80% 120%;
+  background-size: 90% 120%;
 }
 
-.logo {
-  height: 40%;
+.app-logo {
```

Comment:
This is interesting to see how it will look in all the pages. This is just more of a note for myself.